### PR TITLE
ROC-3027: Fix save claim to CCD 

### PR DESCRIPTION
**Pull request by sabahirfan**
### JIRA link (if applicable) ###
[ROC-3027](https://tools.hmcts.net/jira/browse/ROC-3027)

### Change description ###
This PR fixes the issue occurred during merge. Fix generates the correct JSON data.

**Does this PR introduce a breaking change?** (check one with "x")

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/CoreCaseDataService.java [hunk 1]
```diff
@@ -62,7 +66,7 @@ public CaseDetails save(String authorisation, Claim claim) {
                 .save(
                     authorisation,
                     eventRequestData,
-                    jsonMapper.toJson(ccdCase),
+                    toJson(ccdCase),
```
Comment (rlewan) [inline src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/CoreCaseDataService.java]: Why can't we use `JsonMapper` @sabahirfan ?

Comment (sabahirfan) [reply src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/CoreCaseDataService.java]: `jsonMapper` returns String object and if we pass `String` to `CCD Client` we are getting `400` Bad Request.

Comment (rlewan) [reply src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/CoreCaseDataService.java]: Have you investigated why there's a 400 response? It should work with the `String`.

Comment (sabahirfan) [reply src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/CoreCaseDataService.java]: Looks like it was due to double serialisation causing this by generating escaped characters.

**Pull request merged**